scripts/install.sh

```
@@ -173,6 +173,9 @@ download_from_github() {
   local name=$3
   local install_dir=$4
 
+  # Clean version (remove 'v' prefix if present)
+  local clean_version="${release#v}"
```

Shellcheck (0.11.0)

[warning] clean_version appears unused. Verify use (or export if used externally).

(SC2034)